services/api/src/middleware.py

```diff
@@ -76,7 +68,7 @@ def get_user_role(token):
     "/admin-user": False,
     "/admin-new-org": False,
     "/admin-org": False,
-    "/rsu-geo-query": True,
+    "/rsu-geo-query": False,
```

Comment:
We should probably rename this endpoint since it is really not clear what it is. I believe this is the V2X message visualizer, right?

Reply:
Re-named to rsu-config-geo-query